src/_utilities.sass

```diff
@@ -172,4 +172,23 @@
       display: block !important
 
     .frow.visible-print
-      display: flex !important
\ No newline at end of file
+      display: flex !important
+
+  // Accessibility rules
+  // -------------------------
+  .sr-only,
+  .sr-only-focusable:not(:focus)
```

Comment:
Can these be renamed to `.visible-sr` and `.visible-sr-focusable`? That would bring them more inline with `.visible-print` and `.visible-xs` classes for example.

It also would open up the option for future class of `.hidden-sr`. Not sure if that would ever be useful, but I could see it maybe being clever to be able to hide certain content from screenreaders.